content/en/docs/2024.3/Guides/upgrade-cortex/2024.1-to-2024.3/multiple-server-with-ha/upgrade-application-and-loadbalancer-servers.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Upgrade Application Servers and Load Balancer"
+linkTitle: "Upgrade Application Servers and Load Balancer"
+description: "Information about upgrading the Application Servers and Load Balancer Server."
+weight: 30
+---
+
+# {{< param title >}}
+
+This guide describes how to upgrade the Application Servers and Load Balancer Server from 2024.1 to 2024.3.
+
+## Make Installation Artefacts Available
+
+1. Choose one of the Application Servers to be used for to perform the upgrade, and copy the following artefacts to a folder on it:
```

Comment:
Choose one of the Application Servers to be used to perform the upgrade, and copy the following artefacts to a folder on it: